tests/lib/rules/implicit-arrow-linebreak.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * @fileoverview enforce the location of arrow function bodies
+ * @author Sharmila Jesupaul
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/implicit-arrow-linebreak");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+const EXPECTED_LINEBREAK = { message: "Expected a linebreak before this expression." };
+const UNEXPECTED_LINEBREAK = { message: "Expected no linebreak before this expression." };
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 6 } });
+
+ruleTester.run("arrow-linebreak", rule, {
```

Comment:
The rule has been renamed to `implicit-arrow-linebreak`, but this line still calls it `arrow-linebreak`.